src/DocumentationBrowserViewExtension/DocumentationBrowserView.xaml.cs

```diff
@@ -26,6 +26,8 @@ public partial class DocumentationBrowserView : UserControl, IDisposable
 
         internal string WebBrowserUserDataFolder { get; set; }
         internal string FallbackDirectoryName { get; set; }
+        //This folder will be used to store all the images located in Degub/en-US/fallback_docs so we don't need to copy all the images per each language
+        internal const string FallbackImagesDirectoryName = "fallback_docs_images";
```

Comment:
Given this is going to be a new folder in Dynamo delivery, can we call it something like "NodeHelpDocsImages"?

Reply:
changed to NodeHelpDocsImages, see the next commit: https://github.com/DynamoDS/Dynamo/pull/14429/commits/1de8ff2d0a60760996ffe6f98a44835c7cf286e1

Reply:
With the latest decision, would you update the folder to be `NodeHelpSharedDocs` since it would no longer only images

Reply:
updated to NodeHelpSharedDocs, see the next commit:
https://github.com/DynamoDS/Dynamo/pull/14429/commits/ef88df024b2dee733e26ff26f7c81188bdab999f